Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -241,16 +238,18 @@ public void GetAllRevitParamsAndIds(Base speckleElement, DB.Element revitElement
     //sort by key
     foreach (var kv in allParams.OrderBy(x => x.Key))
     {
-#pragma warning disable CA1031 // Do not catch general exception types
       try
       {
         paramBase[kv.Key] = kv.Value;
       }
-      catch
+      catch (InvalidPropNameException)
+      {
+        //ignore
+      }
+      catch (SpeckleException)
       {
         //ignore
       }
```

Comment:
Maybe we should add some logs here, specially if we're swallowing the exception.

In an ideal world failing to get "some" parameters should be reported to the user, but this is not as straightforward and somewhat out of scope for this ticket.

A log call would do here just fine.

Reply:
Added log call in the "SpeckleException" catch. Do we also want one in the InvalidPropName catch? This is thrown whenever we try to assign a dynamic property with invalid characters (e.g. var x = base["hello-world"]) and this happens all the time for these properties.

Reply:
Ok if this happens all the time, then maybe logging `InvalidPropName` is a bit overkill.

The log in speckle exception is good enough 😄